spinn_front_end_common/abstract_models/abstract_rewrites_data_specification.py

```diff
@@ -23,7 +26,18 @@ class AbstractRewritesDataSpecification(object):
         and so can rewrite the data specification
     """
 
-    __slots__ = ()
+    __slots__ = []
+
+    _WRONG_VERTEX_TYPE_ERROR = (
+        "The vertex {} is not of type MachineVertex. By not being a "
+        "machine vertex, the SpiNNFrontEndCommon function DSGRegionReloader "
+        "will not check this vertex")
+
+    def __new__(cls, *more):
```

Comment:
This should be def __new__(cls, *args, **kwargs):

Then it safely covers all cases,

While kwargs is not the spinnaker way of working it is correct in this case

Reply:
fixed